README.md

```diff
@@ -61,6 +61,27 @@ Start MCP server with `sse` transport.
 CRATEDB_MCP_TRANSPORT=sse uv run cratedb-mcp
 ```
 
+# Simple Claude configuration
+To use the MCP version within Claude Desktop, you can use the following configuration:
+
+```json
+{
+  "mcpServers": {
+    "my_cratedb": {
+      "command": "uvx",
+      "args": ["--python", "3.10", "--from", "git+https://github.com/crate/cratedb-mcp", "cratedb-mcp"],
```

Comment:
Maybe we can run a release to be able to compress that into `uvx cratedb-mcp`?

Reply:
I have created https://github.com/crate/cratedb-mcp/issues/5 to track it